### DISCUSSION TITLE
NFS/ Could not parse output

### DISCUSSION THREAD

**Discussion by JML417** [ltsp/ltsp#349] [Q&A] [answered]
Hi, We have been using LTSP for a long time and now wanted to renew our setup so we bought new Lenovo Thinkcentre Clients.
We enabled PXE boot in the BIOS and tried to start but it doesn't work. The IPXE Menu opens without problems and continues to boot but at some point the boot fails.
![IMG_1815 2](https://user-images.githubusercontent.com/38074484/101983006-24701780-3c78-11eb-86a4-97766f727ccf.jpg)
After this error Message continues for some time it says:
"Could not parse output of: ip -o route get 192.168.67.1"
And then booting stops. It is strange because on the old clients everything works fine. 
Is there someone with a similar Issue and a solution to this problem?

**Comment by alkisg** [comment, marked as answer]
Hi, this sounds like a missing network card driver; boot the client with a live USB stick, run the following command and paste the output. Substitute enp0s3 with the actual interface name.

```shell
/usr/lib/klibc/bin/ipconfig -n enp0s3
```

It should show something like this:

```shell
IP-Config: enp0s3 hardware address 08:00:27:65:04:b0 mtu 1500 DHCP RARP
IP-Config: enp0s3 complete (dhcp from 192.168.67.1):
 address: 192.168.67.72    broadcast: 192.168.67.255   netmask: 255.255.255.0   
 gateway: 192.168.67.1     dns0     : 192.168.67.1     dns1   : 194.63.238.4    
 rootserver: 192.168.67.1 rootpath: 
 filename  : 
```

**Comment by JML417** [comment]
Hi, thank you for your answer, the problem was solved by installing the driver of the network card.
